lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -335,6 +342,25 @@ export class SearchFacetFiltersService implements OnDestroy {
         };
     }
 
+    private loadCategoryNames(bucketList: FacetFieldBucket[]) {
+        bucketList.forEach((item) => {
+            const categoryId = item.label.split('/').pop();
+            this.categoryService.getCategory(categoryId)
```

Comment:
It is in forEach so I guess it will be called multiple times? Can't we have some API which takes list of ids at once and make only single call instead of multiple to backend?

Reply:
I don't think we have such an API for now